src/utils/inferencedata.jl

```diff
@@ -1,17 +1,83 @@
-using InferenceObjects
-using PosteriorDB
+function inferencedata(m::SampleModel; kwargs...)
 
-import Base: convert
+    # Used keyword arguments
+    used_keywords = [:include_log_likelihood, :include_sample_stats,
```

Comment:
Why use this design instead of specifying these as default `kwargs` above, e.g.

```julia
function inferencedata(
    m::SampleModel;
    include_log_likelihood=false,
    include_sample_stats=false, 
    include_posterior_predictive=false,
    kwargs...
)
...
```